src/devices/Mcp23xxx/OpCode.cs

```diff
@@ -8,7 +8,7 @@ public class OpCode
     {
         public static byte GetOpCode(int deviceAddress, bool isReadCommand)
         {
-            int opCode = 0b0100_0000 | (deviceAddress << 1);
+            int opCode = deviceAddress << 1;
```

Comment:
I know it is not from this PR but should this class be static given that it only has one static method.

Reply:
You are correct sir.  Thx